### PR TITLE
add is_object before check exsist invoke method in variable

### PR DESCRIPTION
**Pull request by WildZero**
Issue https://github.com/nrk/predis/issues/257

### CODE REVIEW

File: src/Configuration/Options.php [hunk 1]
```diff
@@ -100,8 +100,10 @@ public function __get($option)
             $value = $this->input[$option];
             unset($this->input[$option]);
 
-            if (method_exists($value, '__invoke')) {
-                $value = $value($this, $option);
+            if (is_object($value)) {
+                if (method_exists($value, '__invoke')) {
```
Comment (xabbuh) [inline src/Configuration/Options.php]: Why not simply `is_object($value) && method_exists($value, '__invoke'))`?

Comment (WildZero) [inline src/Configuration/Options.php]: It's my habit: separate different logical condition.

Comment (GrahamCampbell) [inline src/Configuration/Options.php]: Yeh, this should be `is_object($value) && method_exists($value, '__invoke'))`.

**Comment by WildZero** [comment]
I'v fixed it

**Comment by nrk** [comment]
Isn't this one just a duplicate of the already open PR #258?

**Comment by GrahamCampbell** [comment]
Yeh, this is a duplicate.

**Pull request closed without merging**